R/utils.R

```diff
@@ -493,6 +493,30 @@ shinyCallingHandlers <- function(expr) {
   )
 }
 
+shinyUserErrorUnhandled <- function(error, handler = NULL) {
```

Comment:
In what situation would a function be passed in for `handler`?

Reply:
For testing? It's not used currently. It's here as a defensive mechanism; I generally try to avoid having function inputs that are completely controlled by options (c.f. [Make inputs explicit](https://design.tidyverse.org/inputs-explicit.html)).